lib/livebook/text/js.ex

```diff
@@ -0,0 +1,122 @@
+defmodule Livebook.Text.JS do
```

Comment:
This module is fun.

Previously, whenever we were applying a client delta on the server, we would convert the string to a list of UTF16 code units, apply delta to this list, and convert back. Now we work directly on the string binaries.